PyRDF/RDataFrame.py

```diff
@@ -78,6 +78,31 @@ def _get_vector_from_list(self, arg):
 
         return reqd_vec
 
+    def get_num_entries(self):
+        """
+        Gets the number of entries in
+        the given dataset.
+
+        """
+        if isinstance(self.args[0], int):
+            # If there's only one argument
+            # which is an integer, return it.
+            return self.args[0]
+
+        # Construct a ROOT.TChain object
+        import ROOT
```

Comment:
We do not need this import here, it is imported at the beginning of this module.